### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Request Prow to automatically lint any go code in this PR:

/lint

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/knative-operator/commit/7ffc764b0c92e2a4dfa983a8f2e153db7ea06188#commitcomment-43154248

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/knative-operator/commit/b6dc4955cf3d2187494b328910082b6a9c558c24

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

## Proposed Changes

* spelling

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: pkg/client/injection/reconciler/operator/v1alpha1/knativeeventing/reconciler.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func (r *reconcilerImpl) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
 	// Initialize the reconciler state. This will convert the namespace/name
-	// string into a distinct namespace and name, determin if this instance of
+	// string into a distinct namespace and name, determine if this instance of
```
[the remark this answers is not in the document]
Comment (jsoref) [reply pkg/client/injection/reconciler/operator/v1alpha1/knativeeventing/reconciler.go:182]: Can someone please point to the upstream for this?

[the remark this answers is not in the document]
Comment (jsoref) [reply pkg/client/injection/reconciler/operator/v1alpha1/knativeeventing/reconciler.go]: https://github.com/knative/pkg/pull/1797

**Comment by houshengbo** [comment]
@jsoref PLZ join the knative community! Welcome.
could u rebase the pr? not sure why it failed on the builds.

**Review by antoineco — APPROVED** [review]
/lgtm

**Comment by antoineco** [comment]
@jsoref your PR touches some strings inside `_example` values of ConfigMaps. This won't work without updating the corresponding hash. Let me try to help with that.

File: cmd/operator/kodata/knative-serving/0.17.3/2-serving-core.yaml [hunk 2]
```diff
@@ -1138,7 +1138,7 @@ data:
     logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.Response.Code}}, "responseSize": "{{.Response.Size}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
 
     # If true, the request logging will be enabled.
-    # NB: up to and including Knative version 0.18 if logging.requst-log-template is non-empty, this value
+    # NB: up to and including Knative version 0.18 if logging.request-log-template is non-empty, this value
```
Comment (markusthoemmes) [inline cmd/operator/kodata/knative-serving/0.17.3/2-serving-core.yaml:1141]: Ack with @antoineco's comments. These are files pulled in from other upstream projects. Please undo all changes to `cmd/operator/kodata`

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**